### ISSUE TITLE
Problem with migrate in terraform

### ISSUE THREAD

**Issue by dougnd** [google/exposure-notifications-server#627]
### TL;DR
I'm having trouble deploying using terraform. It fails on the migrate step with a password error. I was able to do the terraform step at least once before.  Upon further investigation, one recent related change was #616. My theory is that PR has a bug.

**Expected behavior**
When running `terraform apply` you successfully deploy the server environments.

**Observed behavior**
```
Step #1 - "migrate": Status: Downloaded newer image for gcr.io/cloud-devrel-public-resources/exposure-notifications/migrate:latest
Step #1 - "migrate": gcr.io/cloud-devrel-public-resources/exposure-notifications/migrate:latest
Step #1 - "migrate": ==> Configuring proxy
Step #1 - "migrate": 2020/06/15 01:35:33 Cloud SQL Proxy logging has been disabled by the -quiet flag. All messages (including errors) will be suppressed.
Step #1 - "migrate": ==> Running migrations
Step #1 - "migrate": error: pq: password authentication failed for user "notification"
Finished Step #1 - "migrate"
ERROR
ERROR: build step 1 "gcr.io/cloud-devrel-public-resources/exposure-notifications/migrate:latest" failed: step exited with non-zero status: 1
```

### Reproduction

1. Follow the steps in the [Terraform deploy guide](https://github.com/google/exposure-notifications-server/tree/master/terraform).
2. Create a bucket for terraform state.
3. No generate service.
4. Smaller DB tier:
```
cloudsql_tier         = "db-custom-1-3840"
cloudsql_disk_size_gb = "64"
```
5. Decrease max_connections to 10000. (See [a related comment.](https://github.com/google/exposure-notifications-server/issues/542#issuecomment-640958609)). I believe I also tried to deploy with a larger DB instance that did not require the max connection fix.

**Additional information**

So, the reason I'm looking at #616 is that I tried that urlencode script locally and it didn't work as I expected.  It seems to be used like: `cat ./dbpass | urlencode`. `dbpass` comes from a gcloud secrets dump to file and has no new lines.  The urlencode script has the following:
```
set -eEuo pipefail

read IN
... more code
```
Which, if I'm not mistaken will always exit early with an input without newlines because `set -e` will return early on any non-zero exit code and `read` will return a non-zero exit when it hits EOF (which it will since there is no new line).  Locally, running `echo -n "hi" | ./urlencode` produces no output.

Unless I'm missing something?  It's hard for me to test since I think the migrate run in terraform uses a version from your registry.

Thanks!
